### ISSUE TITLE
PayGO

### ISSUE THREAD

**Issue by leandrossantos** [SistemaFusion/Tef#5]
O TEF da PayGO é usando a DLL deles ou troca de arquivos??

**Comment by robertorp** [comment]
Não, é usado a troca de arquivo, TEF Dial, no momento o framework não tem suporte com DLLs , se alguém ter disposição de me ensinar a usar a DLL é testar para min eu faço a implementação.

**Comment by AndreVasconcellosNogarotto** [comment]
Boa tarde. Saberia me dizer como faço para iniciar o GP TefDial ? 
![tes](https://github.com/SistemaFusion/Tef/assets/24467863/3262a26f-e7f0-4f20-b58e-31314f6651ef)
